common/arrow/src/lib.rs

```diff
@@ -15,6 +15,7 @@
 mod parquet_read;
 mod parquet_write;
 
+pub type ArrayRef = Box<dyn ::arrow::array::Array>;
```

Comment:
How about moving this type def to the back

Reply:
"to the back"? does it mean to define the `ArrayRef` as `Arc<dyn Array>`? 

I am afraid that would bring more troubles:

-  since related methods/functions of `arrow2` are switched from `Arc<_>` to `Box<_>`
-  and generally speaking, if `Box` is applicable(and rustc allows:-), `Arc<_>` should not be used

Reply:
Oh, I mean move to the bottom of file

Reply:
sure!